conans/test/build_helpers/cmake_test.py

```diff
@@ -962,7 +963,8 @@ def test_cmake_system_version_android(self):
             cmake = CMake(conan_file)
             self.assertEquals(cmake.definitions["CMAKE_SYSTEM_VERSION"], "32")
 
-    def test_cmake_system_version_osx(self):
+    @mock.patch('platform.system', return_value="Macos")
```

Comment:
In general, I have some aversion to the mocking and especially with system-wide things. It SHOULD work but I'm always afraid parallelism of tests or stuff like that could affect. I would be ok if the test only runs in Macos